### DISCUSSION TITLE
qFlipper Icon Please!

### DISCUSSION THREAD

**Discussion by dropalltables** [elrumo/macOS_Big_Sur_icons_replacements#1475] [Icon request]
[App download](https://flipperzero.one/update)
Current icon:
<img width="122" alt="Screenshot 2023-12-19 at 10 52 13 AM" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/80474621/00e75a8f-5c39-41c3-9eb6-960f42fedda2">
Improvements:
- remove outside stuff
- make the dolphin 3D

If anyone can do this, THANK YOU!!!! 🙏🙏

**Comment by mendelGith-dev** [comment]
Hello

**Reply by dropalltables** [reply]
Hi!
